docs/image/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu-debootstrap:14.04
+
+RUN apt-get update && apt-get install --yes python python-pip postgresql-9.3 make postgresql-server-dev-9.3  python-dev libffi-dev git  && apt-get clean
```

Comment:
@lorieri we removed the need for PostgreSQL in #2946, so you can simplify these requirements.